include/uxr/agent/transport/discovery/DiscoveryServerLinux.hpp

```diff
@@ -28,27 +29,19 @@ namespace uxr {
 
 class Processor;
 
-class DiscoveryServer
+class DiscoveryServerLinux : public DiscoveryServer
 {
 public:
-    DiscoveryServer(const Processor& processor, uint16_t port, uint16_t discovery_port);
-    ~DiscoveryServer() = default;
-
-    bool run();
-    bool stop();
+    DiscoveryServerLinux(const Processor& processor, uint16_t port, uint16_t discovery_port);
+    ~DiscoveryServerLinux() override = default;
 
 private:
-    bool init();
-    bool close();
-    bool recv_message(InputPacket& input_packet, int timeout);
-    bool send_message(OutputPacket output_packet);
-    void discovery_loop();
+    bool init() override;
+    bool close() override;
+    bool recv_message(InputPacket& input_packet, int timeout) override;
+    bool send_message(OutputPacket output_packet) override;
```

Comment:
It is intended to pass OutputPacket by value?

Reply:
Nop, in fact, I change it by `std::move` in order to gain efficiency.